docs/source/index.rst

```diff
@@ -0,0 +1,21 @@
+Cookiecutter Data Science
```

Comment:
This isn't a data science project cookiecutter really, it's to create a Python package for PyPI. 
```suggestion
Neuroinformatics Unit Python cookiecutter
```